app/forms/waste_carriers_engine/renewal_complete_form.rb

```diff
@@ -11,9 +11,9 @@ def self.can_navigate_flexibly?
     def initialize(transient_registration)
       super
       self.certificate_link = build_certificate_link
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.certificate_link = build_certificate_link
```